discovery/daily_notes/2025-08-22.md

```
@@ -32,3 +32,22 @@ e.g., an avatar that can be pinned to a browser and it can accompany people in r
+I think what I'll do is make it fully interactive, with text + voice input.
+- Something like what Mercor uses in their virtual interviews, bruh.
+
+9:19am: I think I'll actually just piggyback off my existing avatar generation and just change the text passed to ElevenLabs, nice!
+
+Then I think a V2 MVP is to add another profile to the /generate-avatar (just Parker again), but make it a "sports persona" where we can either (1) ask it what's the latest sports news (e.g., what happened in the latest X game) and/or (2) paste in a link to an ESPN game and the avatar will talk about it. A V3 MVP is adding audio integration, so you can "talk" to the avatar.
+
+Ironically I think they want something like Cluely, but for sports.
+
+This is what I sent to the team:
+"""text
+Here's an app that I think sounds similar to the end state that you're looking for, something like a real-time avatar to provide, for example, commentary: https://cluely.com/
```

markdownlint-cli2 (0.17.2)

Bare URL used

(MD034, no-bare-urls)